web/src/hooks/useOnClickOutside.ts

```diff
@@ -12,11 +12,11 @@ const useOnClickOutside = (ref: { current: HTMLElement | null }, callback: () =>
     }
 
     const handleKeyPress = (event: KeyboardEvent) => {
-      const KEY_ENTER = 13
-      const KEY_ESCAPE = 27
-      if (event.keyCode === KEY_ESCAPE) {
+      const KEY_ENTER = 'Enter'
```

Comment:
I think now these constants are not needed any longer, as the naming is already clear.

Reply:
done